docs/generated/packages/angular/executors/dev-server.json

```diff
@@ -109,6 +109,14 @@
         "type": "boolean",
         "description": "Read buildable libraries from source instead of building them separately. If not set, it will take the value specified in the `browserTarget` options, or it will default to `true` if it's also not set in the `browserTarget` options.",
         "x-priority": "important"
+      },
+      "esbuildMiddleware": {
```

Comment:
was not completely sure about the property name, "middleware" seems to be too generic since this executor can be used with both webpack and esbuild

Reply:
That sounds good to me 👍🏻